operators.go

```diff
@@ -78,6 +91,126 @@ func operatorContainsFn(uValue interface{}, cValue interface{}) bool {
 	return false
 }
 
+func operatorLessThanFn(uValue interface{}, cValue interface{}) bool {
+	//TODO: Allow all kinds of numbers? or just float64 like this:
```

Comment:
The parsed cValue will always give us a flaot64 since it came from json, but what about uValue? Sdk users could set it to an integer and still reasonably expect it gets compared, but instead it would just return false.